### ISSUE TITLE
Podman doesn't always honor default rw permission for volumes in podman run

### ISSUE THREAD

**Issue by mandre** [projectatomic/libpod#1129]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
>
> /kind feature

**Description**

Podman does not always honor the default of `:rw`  when mounting a volume with `podman run --volume [...]`. It turns out podman takes the last explicitly specified mount permission as the default.

The doc for `podman run` says "By default, the volumes are mounted read-write".

**Steps to reproduce the issue:**

```
$ sudo podman run --name rabbitmq_bootstrap -ti --user root --rm \
    --volume=/etc/hosts:/etc/hosts:ro \
    --volume=/var/lib/rabbitmq:/var/lib/rabbitmq \
    192.168.24.1:8787/tripleomaster/centos-binary-rabbitmq:current-tripleo \
    touch /var/lib/rabbitmq/foo
touch: cannot touch '/var/lib/rabbitmq/foo': Read-only file system
```

**Describe the results you received:**

It fails to write to `/var/lib/rabbitmq` even though the volume was mounted not supposed to be mounted read-only.

**Describe the results you expected:**

It should be able write to `/var/lib/rabbitmq` when mounting the volume with `--volume=/var/lib/rabbitmq:/var/lib/rabbitmq`

**Additional information you deem important (e.g. issue happens only occasionally):**

Some more tests show that podman uses the last explicitly set mount permission:

```
# WORKS
$ sudo podman run --name rabbitmq_bootstrap -ti --user root --rm \
    --volume=/var/lib/rabbitmq:/var/lib/rabbitmq \
    192.168.24.1:8787/tripleomaster/centos-binary-rabbitmq:current-tripleo \
    touch /var/lib/rabbitmq/foo

# WORKS
$ sudo podman run --name rabbitmq_bootstrap -ti --user root --rm \
    --volume=/etc/hosts:/etc/hosts:ro \
    --volume=/var/lib/rabbitmq:/var/lib/rabbitmq:rw \
    192.168.24.1:8787/tripleomaster/centos-binary-rabbitmq:current-tripleo \
    touch /var/lib/rabbitmq/foo

# FAILS
$ sudo podman run --name rabbitmq_bootstrap -ti --user root --rm \
    --volume=/etc/hosts:/etc/hosts:ro \
    --volume=/etc/resolv.conf:/etc/resolv.conf \
    --volume=/var/lib/rabbitmq:/var/lib/rabbitmq \
    192.168.24.1:8787/tripleomaster/centos-binary-rabbitmq:current-tripleo \
    touch /var/lib/rabbitmq/foo
touch: cannot touch '/var/lib/rabbitmq/foo': Read-only file system

# WORKS
$ sudo podman run --name rabbitmq_bootstrap -ti --user root --rm \
    --volume=/etc/hosts:/etc/hosts:ro \
    --volume=/etc/resolv.conf:/etc/resolv.conf:rw \
    --volume=/var/lib/rabbitmq:/var/lib/rabbitmq \
    192.168.24.1:8787/tripleomaster/centos-binary-rabbitmq:current-tripleo \
    touch /var/lib/rabbitmq/foo
```

**Output of `podman version`:**

```
$ podman version
Version:       0.6.1
Go Version:    go1.9.4
OS/Arch:       linux/amd64
```

**Output of `podman info`:**

```
$ sudo podman info
host:
  MemFree: 261070848
  MemTotal: 8201945088
  SwapFree: 0
  SwapTotal: 0
  arch: amd64
  cpus: 2
  hostname: openstack-controller-0
  kernel: 3.10.0-862.9.1.el7.x86_64
  os: linux
  uptime: 11h 17m 28.52s (Approximately 0.46 days)
insecure registries:
  registries:
  - 192.168.24.1:8787
  - 192.168.24.3:8787
registries:
  registries:
  - docker.io
  - registry.centos.org
  - registry.access.redhat.com
store:
  ContainerStore:
    number: 5
  GraphDriverName: overlay
  GraphOptions:
  - overlay.override_kernel_check=true
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 6
  RunRoot: /var/run/containers/storage
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by mheon** [comment]
Sounds like a parsing issue - @umohnani8 PTAL

**Comment by rhatdan** [comment]
Thanks @mandre This should be an easy fix.

**Comment by umohnani8** [comment]
Fixed in #1142
